src/Components/creatorPage/OtherWorks.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react';
+import propTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import './OtherWorks.css';
+// import { v4 as uuidv4 } from 'uuid';
+
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base(airtableConfig.baseKey);
+
+// Authored and illustrated work components
+function CreatedWorksCard({ authorId }) {
+  const [imageTitleObject, setImageTitleObject] = useState([]);
+  const [illustratedObject, setillustratedObject] = useState([]);
+
+  function FindPosts() {
+    const id = authorId;
+    base('Creator').find(id, (err, records) => {
+      if (err) {
+        return;
+      }
+      const bookid = records.fields.authored;
+      const illustratedId = records.fields.illustrated;
+      if (bookid.length) {
+        bookid.forEach((element) => {
+          base('Book').find(element, (error, record) => {
+            if (error) {
+              return;
+            }
+            setImageTitleObject((prevValue) => prevValue.concat(
+              {
+                image: record.fields.image[0].thumbnails.large.url,
+                title: record.fields.title,
+                id: element,
+              },
+            ));
+          });
+        });
+      }
+      if (illustratedId.length) {
+        illustratedId.forEach((element) => {
+          base('Book').find(element, (error, record) => {
+            if (error) {
+              return;
+            }
+            setillustratedObject((prevValue) => prevValue.concat(
+              {
+                image: record.fields.image[0].thumbnails.large.url,
+                title: record.fields.title,
+                id: element,
+              },
+            ));
+          });
+        });
+      }
+    });
+  }
+
+  useEffect(() => {
+    FindPosts();
+  }, []);
+
+  return (
+    <div>
+      {imageTitleObject.length && <div> Authored Works: </div>}
+      <div>
+        {imageTitleObject.map((element) => (
+          <div key={element.id} className="related-works-card">
```

Comment:
Eventually once the designers have developed more complicated UI for this part, you might want to refactor this code to its own component so that the code doesn't become less legible.